cost/aws/unused_volumes/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v6.1
+
+- Added `resourceName`, `age` and `lookBackPeriod` incident fields
```

Comment:
Like the other CHANGELOG, this should refer to the incident fields by the name the user would see.

Reply:
Updated